src/asyncOperationManagerUtils.js

```diff
@@ -61,7 +61,12 @@ const registerAsyncOperationDescriptors = (asyncOperationDescriptors, ...otherDe
   return asyncOperationManagerState.setState(newState);
 };
 
-const getAsyncOperation = (state, descriptorId, params, otherFields) => {
+const getAsyncOperation = ({
+  state,
+  descriptorId,
+  params,
+  otherFields,
+}) => {
```

Comment:
Is the plan to have a second PR to update all the usages of `getAsyncOperation` in the project?

Reply:
great catch.  I'll change it back to it doesn't need to change throughout the application

Reply:
Oh, ok! Are you going to have `getAsyncOperationFromState` take an object still?

Reply:
yep!  It seems like it wouldn't hurt